tests/Wallabag/CoreBundle/Helper/RedirectTest.php

```diff
@@ -15,21 +15,24 @@ class RedirectTest extends \PHPUnit_Framework_TestCase
     public function setUp()
     {
         $this->routerMock = $this->getRouterMock();
-        $this->redirect = new Redirect($this->routerMock);
+        $tokenStorage = $this->getMockBuilder('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->redirect = new Redirect($this->routerMock, $tokenStorage);
     }
 
     public function testRedirectToNullWithFallback()
     {
         $redirectUrl = $this->redirect->to(null, 'fallback');
 
-        $this->assertEquals('fallback', $redirectUrl);
+        $this->assertEquals(null, $redirectUrl);
```

Comment:
You are wrong when updating these tests. Because here, you are just testing that when a user is null, it returns the given url: `null`.
You are not testing the fallback